python-ecosys/aiohttp/aiohttp/__init__.py

```diff
@@ -263,7 +263,7 @@ def ws_connect(self, url, ssl=None):
         return _WSRequestContextManager(self, self._ws_connect(url, ssl=ssl))
 
     async def _ws_connect(self, url, ssl=None):
-        ws_client = WebSocketClient(None)
+        ws_client = WebSocketClient(self._base_headers)
```

Comment:
I think this should be:
```py
         ws_client = WebSocketClient(self._base_headers.copy())
```
Otherwise the base headers will be mutated for each websocket connection.

Reply:
Do you want to make this change as part of your PR here?